src/test/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/transform/resthandler/RestStartTransformActionIT.kt

```diff
@@ -0,0 +1,189 @@
+import org.elasticsearch.client.ResponseException
+import org.elasticsearch.rest.RestStatus
+import org.elasticsearch.test.junit.annotations.TestLogging
+
+@TestLogging(value = "level:DEBUG", reason = "Debugging tests")
+@Suppress("UNCHECKED_CAST")
+class RestStartTransformActionIT : TransformRestTestCase() {
+
+    @Throws(Exception::class)
+    fun `test starting a stopped transform`() {
+        val transform = createTransform(randomTransform().copy(enabled = false, enabledAt = null, metadataId = null))
+        assertTrue("Transform was not disabled", !transform.enabled)
+
+        val response = client().makeRequest("POST", "$TRANSFORM_BASE_URI/${transform.id}/_start")
+        assertEquals("Start transform failed", RestStatus.OK, response.restStatus())
+        val expectedResponse = mapOf("acknowledged" to true)
+        assertEquals(expectedResponse, response.asMap())
+
+        val updatedTransform = getTransform(transform.id)
+        assertTrue("Transform was not enabled", updatedTransform.enabled)
+    }
+
+    @Throws(Exception::class)
+    fun `test starting a started transform doesnt change enabled time`() {
+        // First create a non-started transform
+        val transform = createTransform(randomTransform().copy(enabled = false, enabledAt = null, metadataId = null))
+        assertTrue("Transform was not disabled", !transform.enabled)
+
+        // Enable it to get the job enabled time
+        val response = client().makeRequest("POST", "$TRANSFORM_BASE_URI/${transform.id}/_start")
+        assertEquals("Start transform failed", RestStatus.OK, response.restStatus())
+        val expectedResponse = mapOf("acknowledged" to true)
+        assertEquals(expectedResponse, response.asMap())
+
+        val updatedTransform = getTransform(transform.id)
+        assertTrue("Transform was not enabled", updatedTransform.enabled)
+
+        val secondResponse = client().makeRequest("POST", "$TRANSFORM_BASE_URI/${transform.id}/_start")
+        assertEquals("Start transform failed", RestStatus.OK, secondResponse.restStatus())
+        val expectedSecondResponse = mapOf("acknowledged" to true)
+        assertEquals(expectedSecondResponse, secondResponse.asMap())
+
+        // Confirm the job enabled time is not reset to a newer time if job was already enabled
+        val updatedSecondTransform = getTransform(transform.id)
+        assertTrue("Transform was not enabled", updatedSecondTransform.enabled)
+        assertEquals("Jobs had different enabled times", updatedTransform.enabledAt, updatedSecondTransform.enabledAt)
+    }
+
+    @Throws(Exception::class)
+    fun `test start a transform with no id fails`() {
+        try {
+            client().makeRequest("POST", "$TRANSFORM_BASE_URI//_start")
+            fail("Expected 400 Method BAD_REQUEST response")
+        } catch (e: ResponseException) {
+            assertEquals("Unexpected status", RestStatus.BAD_REQUEST, e.response.restStatus())
+        }
+    }
+
+    /*
```

Comment:
can you uncomment these? 
Also can you add testcase for attempting to start a transform that doesn't exist?